### PR TITLE
[Bluetooth][Non-ACR] Define Interop callback to global variable

### PR DESCRIPTION
**Pull request by Wootak**
Signed-off-by: Wootak Jung <wootak.jung@samsung.com>

### CODE REVIEW

**Review by dh79pyun — APPROVED** [review]
Looks good.
OPP C# TCT issue is resolved by this patchset

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAdapterImpl.cs [hunk 1]
```diff
@@ -479,7 +480,7 @@ internal IEnumerable<BluetoothDevice> GetBondedDevices()
                 }
                 return true;
             };
-            int ret = Interop.Bluetooth.GetBondedDevices(callback, IntPtr.Zero);
+            int ret = Interop.Bluetooth.GetBondedDevices(_bondedDeviceCallback, IntPtr.Zero);
```
Comment (myroot) [inline src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAdapterImpl.cs:483]: `GetBoundedDevices` is not async method, so need not change.
 anyway it also no matter

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothDevice.cs [hunk 2]
```diff
@@ -619,15 +620,15 @@ public IEnumerable<BluetoothProfileType> GetConnectedProfiles()
...
+                _connectedProfileCallback = (int profile, IntPtr userData) =>
                 {
                     if (!profile.Equals(null))
                     {
                         profileList.Add((BluetoothProfileType)profile);
                     }
                     return true;
                 };
-                int ret = Interop.Bluetooth.GetConnectedProfiles(RemoteDeviceAddress, callback, IntPtr.Zero);
+                int ret = Interop.Bluetooth.GetConnectedProfiles(RemoteDeviceAddress, _connectedProfileCallback, IntPtr.Zero);
```
Comment (myroot) [inline src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothDevice.cs:631]: `GetConnectedProfiles` is not async method, so need not change.
 anyway it also no matter

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs [hunk 3]
```diff
@@ -100,7 +101,7 @@ internal IEnumerable<BluetoothGattService> GetServices(BluetoothGattServer serve
                 return true;
             };
 
-            int err = Interop.Bluetooth.BtGattServerForeachServices(_handle, cb, IntPtr.Zero);
+            int err = Interop.Bluetooth.BtGattServerForeachServices(_handle, _serviceForeachCallback, IntPtr.Zero);
```
Comment (myroot) [inline src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs:104]: `BtGattServerForeachServices` is not async method, so need not change.
anyway it also no matter

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs [hunk 4]
```diff
@@ -234,7 +236,7 @@ internal IEnumerable<BluetoothGattService> GetServices(BluetoothGattClient clien
                 return true;
             };
 
-            int err = Interop.Bluetooth.BtGattClientForeachServices(_handle, cb, IntPtr.Zero);
+            int err = Interop.Bluetooth.BtGattClientForeachServices(_handle, _serviceForeachCallback, IntPtr.Zero);
```
Comment (myroot) [inline src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs:239]: `BtGattClientForeachServices` is not async method, so need not change.
anyway it also no matter

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs [hunk 5]
```diff
@@ -544,7 +551,7 @@ internal IEnumerable<BluetoothGattDescriptor> GetDescriptors(BluetoothGattCharac
                 return true;
             };
 
-            int err = Interop.Bluetooth.BtGattCharacteristicForeachDescriptors(characteristic.GetHandle(), cb, IntPtr.Zero);
+            int err = Interop.Bluetooth.BtGattCharacteristicForeachDescriptors(characteristic.GetHandle(), _descriptorForeachCallback, IntPtr.Zero);
```
Comment (myroot) [inline src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs:554]: `foreach` is not async method, so need not change.
anyway it also no matter

**Review by myroot — APPROVED** [review]

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAdapterImpl.cs (hunk 1 above)
Comment (Wootak) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAdapterImpl.cs:483]: Thank you for comment. I want to leave it as it is.

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothDevice.cs (hunk 2 above)
Comment (Wootak) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothDevice.cs:631]: Thank you for comment. I want to leave it as it is.

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs (hunk 3 above)
Comment (Wootak) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs:104]: Thank you for comment. I want to leave it as it is.

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs (hunk 4 above)
Comment (Wootak) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs:239]: Thank you for comment. I want to leave it as it is.

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs (hunk 5 above)
Comment (Wootak) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs:554]: Thank you for comment. I want to leave it as it is.

**Pull request merged**